### PR TITLE
allow_get removal

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
Fixes according upstream

### CODE REVIEW

File: otcextensions/sdk/anti_ddos/v1/status.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ class TaskStatus(resource.Resource):
 
     # capabilities
     allow_fetch = True
+    allow_get = True
```
Comment (gtema) [inline otcextensions/sdk/anti_ddos/v1/status.py]: In some cases you added still allow_get. Why?

**Review by lego963 — DISMISSED** [review]

**Review by vladimirvshivkov — DISMISSED** [review]

File: otcextensions/sdk/css/v1/snapshot.py [hunk 2]
```diff
@@ -33,6 +33,7 @@ class Snapshot(resource.Resource):
     allow_create = True
     allow_delete = True
     allow_list = True
+    allow_get = True
```
Comment (gtema) [inline otcextensions/sdk/css/v1/snapshot.py]: Here also

File: otcextensions/sdk/kms/v1/key.py [hunk 3]
```diff
@@ -27,6 +27,7 @@ class Key(_base.Resource):
     allow_create = True
     allow_update = False
     allow_fetch = True
+    allow_get = True
```
Comment (gtema) [inline otcextensions/sdk/kms/v1/key.py]: And here

**Review by gtema — CHANGES_REQUESTED** [review]

File: otcextensions/sdk/anti_ddos/v1/status.py (hunk 1 above)
Comment (anton-sidelnikov) [reply otcextensions/sdk/anti_ddos/v1/status.py]: this is strange, yesterday it fails somehow without get. Fixed

**Review by gtema — APPROVED** [review]

**Review by lego963 — APPROVED** [review]

**Pull request merged**